### DISCUSSION TITLE
seems we can release v5.6(1)

### DISCUSSION THREAD

**Discussion by joomod** [slabbi/uddeIM#109] [General]
as there are no issues commented I think we can release v5.6 (along with config 2.9)
changes:
**5.6.** 
- **new config v2.9** 
   (autocomplete in system-tab no more disable, option to add predefined message to new message link)
- edit script for predefined message in url query for new message link
- issue when user is logged out by system loaded pfrontend
- corrected some php8.2 warnings
- added **pkg_uddeim_search** to use the uddeim_search_plugin with com_search

**5.6.1**
changed restore.gif icon and added/edit recall.gif to get different icons in postbox view

I have updated the main with the 5.6.1 changes, but **NOT** the 5.6 changes

so here are the new files which must be added for release 5.6.1

[com_uddeim561_j50.zip](https://github.com/user-attachments/files/17297185/com_uddeim561_j50.zip)
[pkg_uddeim_search_j50.zip](https://github.com/user-attachments/files/17297190/pkg_uddeim_search_j50.zip)

**Comment by slabbi** [comment]
Please upload the file into the "main" branch.
"plug_uddeim_searchbot_j50" was a mess, so I deleted the zip there.

**Reply by joomod** [reply]
how can I upload directly in the main?
(uploaded new zip in com_uddeimj50)
but where to place pkg_search

ok, found.
both files are now in the main

**Comment by slabbi** [comment]
first "git pull origin main", change the files, commit and push these back.
Or provide a pkg_uddeim_search package that I can extract in the folder without having a mixture of installation files and zips.

**Comment by slabbi** [comment]
com_uddeim_j50 has been updated

**Comment by slabbi** [comment]
Please stop uploading zip filers in the main.

**Reply by joomod** [reply]
sorry, I think you asked for ?

**Comment by slabbi** [comment]
added "pkg_uddeim_search_j50"

**Comment by slabbi** [comment]
You can upload zips here but not in the development branch.

**Comment by slabbi** [comment]
You should really get familiar with "git". You can use GIT GUI, https://gitforwindows.org/

**Comment by joomod** [comment]
ok, thank you.
can you now also add the new release?

**Comment by slabbi** [comment]
No, sorry. Not yet.

There is still some work to do:
1. Check consistency of version number and date. For example uddeim.xml as still 1.7.2024 and version 5.5.

2. I have really bad feeling with that:
<updateservers>
	<server type="extension" priority="1" name="Uddeim Update Server">https://raw.githack.com/slabbi/uddeIM/main/uddeim_update.xml</server>
</updateservers>

Is this server trustworthy? Don't think so. Please revert to official Github servers. Otherwise malicious users might deliver a modified update file that points to modified software. Update should also work with Github.

**Comment by slabbi** [comment]
In the meanwhile I update the the build scripts for the installation zips.

**Comment by joomod** [comment]
1. ok, forgot updating the xml file - done
2. raw.githack.com ?? don't know, but it was the only working I found in the beginning
    I changed this to Uddeim Update Site and replaced it by https://raw.githubusercontent.com
but I remember that I have read this is/will be closed
(or do you know a better way ??)

**Reply by slabbi** [reply]
Great.
Well, don't know. We should use a trustworthy platform to deliver code. Actually I do not now s better way.

I googled for more info and found this
https://github.com/neoascetic/rawgithack/issues/55
https://github.com/rgrove/rawgit/issues/197
So we share the same concerns.

**Comment by joomod** [comment]
the main difference is that the old raw.githubusercontent only will open a raw text site,
while the raw.githack opens a real xml.file
I just copied the update link from the joomla com_search, but I don't know if it works

**Comment by slabbi** [comment]
released
